src/sentry/nodestore/base.py

```diff
@@ -58,39 +65,120 @@ def delete_multi(self, id_list):
         for id in id_list:
             self.delete(id)
 
-    def get(self, id):
+    def _decode(self, value, subkey):
+        if value is None:
+            return None
+
+        lines_iter = iter(value.splitlines())
+        try:
+            if subkey is not None:
+                next(lines_iter)
+
+                for line in lines_iter:
+                    if line.strip() == subkey.encode("utf8"):
+                        break
+
+            return json_loads(next(lines_iter))
+        except StopIteration:
+            return None
+
+    def _get_bytes(self, id):
```

Comment:
Add to `__all__`?

Reply:
Will remove other private methods from `__all__` instead.